src/ensembl/tests/fixtures/browser.ts

```diff
@@ -50,36 +50,51 @@ export const createZmenuContent = () => [
         [{ markup: [Markup.LIGHT, Markup.FOCUS], text: faker.lorem.words() }]
       ]
     ],
-    track_id: 'track:gene-feat'
+    track_id: faker.lorem.words()
   }
 ];
 
-export const createValidationInfo = (): RegionValidationResponse => ({
-  end: {
-    error_code: null,
-    error_message: null,
-    is_valid: true,
-    value: 32400266
-  },
-  genome_id: {
-    error_code: null,
-    error_message: null,
-    is_valid: true,
-    value: 'homo_sapiens_GCA_000001405_27'
-  },
-  is_parseable: true,
-  region: {
-    error_code: null,
-    error_message: null,
-    is_valid: true,
-    region_code: 'chromosome',
-    region_name: '13'
-  },
-  region_id: 'homo_sapiens_GCA_000001405_27:region:13:32315086-32400266',
-  start: {
-    error_code: null,
-    error_message: null,
-    is_valid: true,
-    value: 32315086
-  }
-});
+export const createValidationInfo = (): RegionValidationResponse => {
+  const randomValue = Math.random();
+
+  return {
+    end: {
+      error_code: null,
+      error_message: null,
+      is_valid: true,
+      value: randomValue
+    },
+    genome_id: {
+      error_code: null,
+      error_message: null,
+      is_valid: true,
+      value: faker.lorem.words()
+    },
+    is_parseable: true,
+    region: {
+      error_code: null,
+      error_message: null,
+      is_valid: true,
+      region_code: 'chromosome',
+      region_name: '13'
+    },
+    region_id: faker.lorem.words(),
+    start: {
+      error_code: null,
+      error_message: null,
+      is_valid: true,
+      value: randomValue + Math.random()
+    }
+  };
+};
+
+export const createChrLocationValues = () => {
+  const region = faker.lorem.words();
+  const start = faker.random.number();
+  const end = faker.random.number();
+
+  return {
+    tuppleValue: [region, start, end] as ChrLocation,
+    stringValue: `${region}:${start}-${end}`
```

Comment:
Browser helper?

Reply:
I didn't use it as it is not clear which version to use. Should it be `getChrLocationFromStr` that returns a tuple or `getChrLocationStr` that returns a `string`?

Reply:
I would think `getChrLocationStr`.